resources/fonts/658fccc2ea95579d9a08f28d55643893.ufm

```diff
@@ -0,0 +1,239 @@
+StartFontMetrics 4.1
```

Comment:
never heard of `.ufm` assume you need this one and the `.ufm.php` file?

Reply:
It's a font metric file for dompdf, for difficult fonts so you can set dompdf up with some hints.

Reply:
:+1: